server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetFieldMappingAction.java

```diff
@@ -48,9 +48,9 @@
 
     private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
         LogManager.getLogger(RestGetFieldMappingAction.class));
-    public static final String TYPES_DEPRECATION_MESSAGE = "[types removal] The response format of get field " +
-        "mapping requests will change in 7.0. Please start using the include_type_name parameter set to false " +
-        "to move to the new, typeless response format that will become the default.";
+    static final String TYPES_DEPRECATION_MESSAGE = "[types removal] The parameter include_type_name " +
+        "should be explicitly specified in get index requests to prepare for 7.0. In 7.0 include_type_name " +
```

Comment:
> get index requests
@jtibshirani  should it be "get field mapping requests"?

Reply:
Thanks @mayya-sharipova, will fix. I somehow make this mistake every time...